### ISSUE TITLE
Migrations.change/3 will call Migrations.V5 if initial add_oban_jobs_table uses defdelegate

### ISSUE THREAD

**Issue by seanwash** [sorentwo/oban#63]
I believe the installation instructions used to specify an initial migration like so:

```elixir
defmodule MyApp.Migrations.AddObanJobsTable do
  use Ecto.Migration

  defdelegate up, to: Oban.Migrations
  defdelegate down, to: Oban.Migrations
end
```

When running `MIX_ENV=test mix ecto.reset` and I CI I get the following error:

```elixir
** (ArgumentError) argument error
    :erlang.binary_to_existing_atom("Elixir.Oban.Migrations.V5", :utf8)
    lib/oban/migrations.ex:45: anonymous fn/4 in Oban.Migrations.change/3
    (elixir) lib/enum.ex:3011: Enum.reduce_range_dec/4
    lib/oban/migrations.ex:43: Oban.Migrations.change/3
    (ecto_sql) lib/ecto/migration/runner.ex:280: Ecto.Migration.Runner.perform_operation/3
    (stdlib) timer.erl:166: :timer.tc/1
    (ecto_sql) lib/ecto/migration/runner.ex:27: Ecto.Migration.Runner.run/7
```

I noticed that the readme now specifies an initial `add_oban_jobs_table` migration where `up/1` doesn't specify a version opt which results in the same error listed above. Specifying a version like `Oban.Migrations.up(version: 2)` works though.

https://github.com/sorentwo/oban/blob/master/lib/oban/migrations.ex#L12 gets a version of 4 unless specified and the range is incremented by 1 which lands us at `Migrations.V5` on line 45 (I think).

-- edit

Yeah I think I'm wrong about the actual cause, but if I can figure it out soon I'll PR it :D

**Comment by sorentwo** [comment]
This was definitely a problem before f75e5f2 and f944b39, but there are tests for this now: https://github.com/sorentwo/oban/blob/master/test/integration/migrating_test.exs#L65

Are you running the v0.8.0 release, or an earlier version through git?

**Comment by seanwash** [comment]
Ah, I had been running master because of my system/docker times not being synced, now I’m running 0.8.0. After adjusting my initial migration everything was 👍 .

**Comment by sorentwo** [comment]
Great to hear it! I'll close this issue then 👍

**Comment by tcoopman** [comment]
I'm running into the same issue when I start from a clear database and run all migrations. This is on `0.8.0`.

**Comment by tcoopman** [comment]
The fix was to set `version: 2` fixed in the migration.
It might be useful to warn about this in the readme, and it looks to me that it would be a good idea to set a fixed version instead of just `up`. I want both to be able to keep upgrading my db and start from an empty db.

**Comment by sorentwo** [comment]
Ok, _really_ fixed this time with 6972afd. Thanks for the additional details @tcoopman, that made the cause more apparent.

**Comment by victorolinasc** [comment]
Can you release a patch version with this fix?

**Comment by sorentwo** [comment]
@victorolinasc Absolutely! I just released [0.8.1](https://hex.pm/packages/oban/0.8.1).
